src/coreclr/interpreter/ee_il_dll.cpp

```diff
@@ -0,0 +1,142 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+#include <stddef.h>
+#include "corjit.h"
+
+#include "interpreter.h"
+#include "ee_il_dll.hpp"
+
+#include <string.h>
+#include <stdio.h>
+
+#ifdef _MSC_VER
+#define DLLEXPORT
+#else
+#define DLLEXPORT __attribute__ ((visibility ("default")))
+#endif // _MSC_VER
+
+#include <vector>
+
+// FIXME We will probably end up not needing this table.
+// If deemed useful, use some hashtable implementation instead.
+std::vector<std::pair<CORINFO_METHOD_HANDLE,InterpMethod*>> g_interpCodeHash;
+
+static InterpMethod* InterpGetInterpMethod(CORINFO_METHOD_HANDLE methodHnd)
```

Comment:
Are there scenarios where we want to do a get without on-demand allocating a new one? Should this method have a 'create_if_missing' bool param?